scripts/correctionlib_comparison.py

```diff
@@ -0,0 +1,401 @@
+import argparse
+import gzip
+import json
+import os
+from collections import defaultdict
+from itertools import combinations, count, product
+from typing import Any, Dict, Generator, List, Tuple, Union
+
+import matplotlib
+import matplotlib.pyplot as plt
+import mplhep as hep
+import numpy as np
+
+matplotlib.use("Agg")
+hep.style.use("CMS")
+
+parser = argparse.ArgumentParser(description="Plotting comparison of two correction libs")
+parser.add_argument("--input_a", type=str, help="json.gz correctionlib file A")
```

Comment:
--input-a without underscores in the args, same for all other args

Reply:
Added in [0887179](https://github.com/KIT-CMS/TauAnalysis-CROWN/commit/0887179c5d37b3f287597f9342d1aacb647ce35d)

https://github.com/KIT-CMS/TauAnalysis-CROWN/blob/0887179c5d37b3f287597f9342d1aacb647ce35d/scripts/correctionlib_comparison.py#L17-L29